Spanner/tests/System/TransactionTest.php

```diff
@@ -264,6 +265,112 @@ public function testRunTransactionWithDbRole($db, $values, $expected)
         }
     }
 
+    // @TODO: Need to work on this once the directedRead is available
+    /**
+     * @dataProvider getDirectedReadOptions
+     */
+    public function testRWTransactionExecuteWithDirectedReadOptions($directedReadOptions)
+    {
+        $this->markTestSkipped('fixme');
```

Comment:
remove
```suggestion
```

Reply:
Still its not throwing error in my machine!

Reply:
This comment is addressed as Spanner team enabled this feature, please have a look now